Sources/Data/models/WalletUserAsset.swift

```diff
@@ -35,7 +36,7 @@ public final class WalletUserAsset: NSManagedObject, CRUD {
       isErc721: self.isERC721,
       isErc1155: self.isERC1155,
       isNft: self.isNFT,
-      isSpam: false, // TODO: Handle `isSpam` in `WalletUserAsset`
```

Comment:
Any migration necessary ?

Reply:
Nope. It gets set as an non-optional with a default value `false`